AllReadyApp/Web-App/AllReady/Features/Notifications/VolunteerAssignedToItineraryHandler.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AllReady.Models;
+using MediatR;
+using Microsoft.Data.Entity;
+
+namespace AllReady.Features.Notifications
+{
+    public class VolunteerAssignedToItineraryHandler : IAsyncNotificationHandler<VolunteerAssignedToItinerary>
+    {
+        private readonly AllReadyContext _context;
+        private readonly IMediator _mediator;
+
+        public VolunteerAssignedToItineraryHandler(AllReadyContext context, IMediator mediator)
+        {
+            _context = context;
+            _mediator = mediator;
+        }
+
+        public async Task Handle(VolunteerAssignedToItinerary notification)
+        {
+            var taskSignup = await _context.TaskSignups.SingleAsync(ts => ts.Id == notification.TaskSignupId).ConfigureAwait(false);
```

Comment:
@stevejgordon, thanks for the feedback. I still couldn't set up my system to run through an entire smoke test of this handler... I could only get so far, but I still decided to submit the PR so I could share my code. I'll keep trying to do a fresh setup following your instructions (it's probably something I'm missing on my end) so I can get an end to end test on this.

Did you want me to include User and update the PR or do you want to have a crack at it first?